src/js/controllers/mapController.ts

```diff
@@ -213,6 +230,244 @@ export class MapController {
     this._mapview?.graphics.remove(this._previousSketchGraphic);
     this._sketchVM?.create('polygon', { mode: 'freehand' });
   };
+
+  convertSquareMetersToSpecificUnit(
+    distance: number,
+    distanceUnit: string
+  ): number | undefined {
+    let convertedValue;
+    switch (distanceUnit) {
+      case 'acres':
+        convertedValue = distance / 4047;
+        break;
+      case 'square-miles':
+        convertedValue = distance / 2.59e6;
+        break;
+      case 'square-kilometers':
+        convertedValue = distance / 1e6;
+        break;
+      case 'hectares':
+        convertedValue = distance / 10000;
+        break;
+      case 'square-yards':
+        convertedValue = distance * 1.196;
+        break;
+      case 'square-us-feet':
+        convertedValue = distance * 10.764;
+        break;
+      case 'square-meters':
+        convertedValue = distance;
+        break;
+    }
+
+    return convertedValue;
+  }
+
+  convertMetersToSpecificUnit(
+    distance: number,
+    distanceUnit: string
+  ): number | undefined {
+    let convertedValue;
+    switch (distanceUnit) {
+      case 'kilometers':
+        convertedValue = distance / 1000;
+        break;
+      case 'feet':
+      case 'us-feet':
+        convertedValue = distance * 3.281;
+        break;
+      case 'yards':
+        convertedValue = distance * 1.094;
+        break;
+      case 'nautical-miles':
+        convertedValue = distance / 1852;
+        break;
+      case 'meters':
+        convertedValue = distance;
+        break;
+      case 'miles':
+      default:
+        convertedValue = distance / 1609;
+        break;
+    }
+
+    return convertedValue;
+  }
+
+  convertDecimalToDMS(coordinateResults: any): object {
+    const { latitude, longitude } = coordinateResults;
+
+    const latitudeInteger = Math.floor(latitude);
+    const latitudeMinutes = Math.floor((latitude % 1) * 60);
+    const latitudeSeconds = Math.floor((latitudeMinutes % 1) * 60);
+
+    const longitudeInteger = Math.floor(longitude);
+    const longitudeMinutes = Math.floor((longitude % 1) * 60);
+    const longitudeSeconds = Math.floor((longitudeMinutes % 1) * 60);
+
+    const latitudeInDMS = `${latitudeInteger}°${latitudeMinutes}'${latitudeSeconds}"`;
+    const longitudeDMS = `${longitudeInteger}°${longitudeMinutes}'${longitudeSeconds}"`;
```

Comment:
I'll rename either `latitudeInDMS` or `longitudeDMS` during tomorrow's PR review to sync changes into 1 commit